nes/src/nes.cpp

```diff
@@ -44,7 +44,11 @@ class Nes::Impl {
               ppu(PpuFactory::create(&ppu_registers, ppu_mmu.get())),
               mmu(MmuFactory::create(
                       MemBankFactory::create_nes_mem_banks(ppu.get()))),
-              cpu(CpuFactory::create_mos6502(&cpu_registers, mmu.get())) {}
+              cpu(CpuFactory::create_mos6502(&cpu_registers, mmu.get())) {
+        cpu_registers.p = 0x24;
```

Comment:
It might be nicer to assign this by `|`ing together the flags you want?

Reply:
Good idea!